src/shared-render.test.js

```diff
@@ -0,0 +1,93 @@
+const {h, Component} = require('preact');
+
+const {render, shallow} = require('./preact-render-spy');
+
+const shape = (func, funcMember) => {
```

Comment:
I renamed with `sharedTests`. Changed the args to `name, func`. `name` to add something to the test name for debugging failures and `func` to pass the func to use in.